components/ModHealthcareProfessionalSearchbar.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <div>
+        <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
```

Comment:
avoid using custom colors, try using one of our theme variables like `text-primary`, `text-secondary`, or `text-tertiary` or one of their subsets like this:

![image](https://github.com/ourjapanlife/findadoc-web/assets/4411499/a42acb3f-cd0e-4767-b858-187f350bae12)